flexmeasures/cli/data_show.py

```diff
@@ -441,6 +440,7 @@ def chart(
     "sensors",
     required=True,
     multiple=True,
+    callback=validate_unique,
```

Comment:
I think we shouldn't raise whenever the same ID is passed multiple times... Perhaps, It should be a warning and the repeated values shouldn't be considered?